lib/livebook_web/live/session_live.ex

```diff
@@ -77,6 +78,10 @@ defmodule LivebookWeb.SessionLive do
     end)
   end
 
+  defp assign_policy(socket) do
+    assign(socket, policy: %{read: false, execute: false, comment: false, edit: false})
```

Comment:
So the idea is to assign this policy conditionally, so if it's the readonly live action we use this policy, otherwise we use the permissive one.